.github/workflows/test.yml

```diff
@@ -49,8 +49,10 @@ jobs:
         run: pip install -r requirements.txt
       - name: Install collection dependencies
         run: ansible-galaxy install -r requirements.yml
-      - name: Start VM
-        run: vagrant up quadlet
+      - name: Start VMs
+        run: |
+          vagrant up quadlet
+          vagrant up client
```

Comment:
Do you intentionally start them up sequentially instead of parallel? Is that the issue with fetching the same box concurrently giving issues?

Reply:
I honestly haven't tried 😅